internals/daemon/api_warnings_test.go

```diff
@@ -53,7 +53,7 @@ func (s *apiSuite) testWarnings(c *check.C, all bool, body io.Reader) (calls str
 	if all {
 		q.Set("select", "all")
 	}
-	req, err := http.NewRequest(method, "/v2/warnings?"+q.Encode(), body)
+	req, err := http.NewRequest(method, "/v1/warnings?"+q.Encode(), body)
```

Comment:
what's happening here?

Reply:
I believe this was a carry-over from snapd, which uses /v2/ for most API prefixes, and this was just forgotten when moving to Pebble. It didn't actually make the test fail, because of how we use `apiCmd()` above.